specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/EventHubCluster.json

```diff
@@ -0,0 +1,591 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "EventHubClusterManagementClient",
+    "description": "Azure Event Hubs Cluster client for managing Event Hubs Cluster resources.",
+    "version": "2018-01-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EventHub/clusters": {
+      "get": {
+        "tags": [
+          "Event Hubs Clusters"
+        ],
+        "operationId": "EHClusters_ListByResourceGroup",
```

Comment:
Why does this operation group have the "EH" prefix?  All the others are simply "Clusters".  IMO it should be "Clusters" like the rest.